packages/amplify-graphql-transformer/src/graphql-transformer.ts

```diff
@@ -118,6 +119,7 @@ export type ExecuteTransformConfig = TransformConfig &
     parameterProvider?: TransformParameterProvider;
     assetProvider: AssetProvider;
     synthParameters: SynthParameters;
+    deploymentIdentifier?: DeploymentIdentifier;
```

Comment:
This file only applies to Gen 1 transformer right? Is `deploymentIdentifier` required for Gen 1? (Or I might be misremembering how this file is used!)

Reply:
This file is used in both Gen 1 and Gen 2. But `deploymentIdentifier` is not required for Gen 1